### PR TITLE
Change readme

### PR DESCRIPTION
**Pull request by nvdaes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # l10nTest
 Test for the add-ons system
+
+Changing readme.
```
Comment (mhameed) [inline README.md:4]: need to add author fieldn

**Comment by mhameed** [comment]
This uses correct markup, so good to go.

**Comment by nvdaes** [comment]
Thanks seen
:)

**Review by mhameed — CHANGES_REQUESTED** [review]
please consider adding author and license

**Pull request merged**